packages/grafana-data/src/types/jest/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Observable } from 'rxjs';
```

Comment:
Would it make sense to not to duplicate this file and just include the typing from public app in the tsconfig.json?

Reply:
I don't think we want dependencies on our main app from grafana-data, but otherwise I agree.

Reply:
It is the other way around but I agree. However we already do this 😄  if you look at the tsconfig.json in grafana-data it includes types from public app. I think this is not a problem because these are only meant for testing.

Reply:
The best solution should be to have these jest.matchers in a completely separate package but instead I made the main app depend on typings in grafana-data. That makes the most sense to me at least.